core/src/main/java/com/sequenceiq/cloudbreak/service/cluster/ClusterService.java

```diff
@@ -253,6 +263,12 @@ public Cluster create(IdentityUser user, Stack stack, Cluster cluster, List<Clus
         });
     }
 
+    private void setOrganization(Stack stack, Cluster cluster) {
```

Comment:
The name suggest it will set organization for both, but only for `Cluster`

Reply:
I'll rename it